backend/core/src/assets/assets.controller.ts

```diff
@@ -1,11 +1,18 @@
 import {
+  BadRequestException,
   Body,
   Controller,
   Get,
+  NotFoundException,
```

Comment:
Can you remove this since it's not used?

Reply:
Yep, removed.  I think it got imported automatically with one way I was handling errors but never got removed after